oqs_template/generate.yml

```diff
@@ -176,290 +133,98 @@ sigs:
-    claimed_security_level: '5'
-  -
-    family: 'SPHINCS-Haraka'
-    name: 'sphincsharaka256ssimple'
-    oid: '1 3 9999 6 3 7'
-    oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x03, 0x07'
-    oqs_meth: 'OQS_SIG_alg_sphincs_haraka_256s_simple'
-    code_point: '0xfe5c'
-    claimed_security_level: '5'
-  -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256128frobust'
-    oid: '1 3 9999 6 4 1'
-    oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x04, 0x01'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_128f_robust'
-    code_point: '0xfe5e'
-    claimed_security_level: '1'
-  -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256128fsimple'
+    family: 'SPHINCS-SHA2'
+    name: 'sphincssha2128fsimple'
     oid: '1 3 9999 6 4 4'
     oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x04, 0x04'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_128f_simple'
-    code_point: '0xfe61'
-    claimed_security_level: '1'
-  -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256128srobust'
-    oid: '1 3 9999 6 4 7'
-    oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x04, 0x07'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_128s_robust'
-    code_point: '0xfe64'
+    oqs_meth: 'OQS_SIG_alg_sphincs_sha2_128f_simple'
+    code_point: '0xfeb3'
     claimed_security_level: '1'
   -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256128ssimple'
+    family: 'SPHINCS-SHA2'
+    name: 'sphincssha2128ssimple'
     oid: '1 3 9999 6 4 10'
     oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x04, 0x0A'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_128s_simple'
-    code_point: '0xfe67'
+    oqs_meth: 'OQS_SIG_alg_sphincs_sha2_128s_simple'
+    code_point: '0xfeb6'
     claimed_security_level: '1'
   -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256192frobust'
-    oid: '1 3 9999 6 5 1'
-    oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x05, 0x01'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_192f_robust'
-    code_point: '0xfe6a'
-    claimed_security_level: '3'
-  -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256192fsimple'
+    family: 'SPHINCS-SHA2'
+    name: 'sphincssha2192fsimple'
     oid: '1 3 9999 6 5 3'
     oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x05, 0x03'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_192f_simple'
-    code_point: '0xfe6c'
-    claimed_security_level: '3'
-  -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256192srobust'
-    oid: '1 3 9999 6 5 5'
-    oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x05, 0x05'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_192s_robust'
-    code_point: '0xfe6e'
+    oqs_meth: 'OQS_SIG_alg_sphincs_sha2_192f_simple'
+    code_point: '0xfeb9'
     claimed_security_level: '3'
   -
-    family: 'SPHINCS-SHA256'
-    name: 'sphincssha256192ssimple'
+    family: 'SPHINCS-SHA2'
+    name: 'sphincssha2192ssimple'
     oid: '1 3 9999 6 5 7'
     oid_encoded: '0x2B, 0xCE, 0x0F, 0x06, 0x05, 0x07'
-    oqs_meth: 'OQS_SIG_alg_sphincs_sha256_192s_simple'
-    code_point: '0xfe70'
+    oqs_meth: 'OQS_SIG_alg_sphincs_sha2_192s_simple'
+    code_point: '0xfebb'
```

Comment:
Code point is in sync with oqsprovider, but OID is not: Intentional? Irrelevant? Is this getting interop-tested somehow (against what?)?

Reply:
I fixed the OIDs, and as far as I know, we haven't done interop testing in a while have we? I can add it back to BoringSSL if need be.

Reply:
> add it back to BoringSSL if need be.

I'd say it only needs to be if we want to keep supporting BoringSSL and Chromium. Do we? @dstebila ?@christianpaquin?

Reply:
If I remember our last discussion correctly: we can support it on a best-effort basis; i.e., fix things when we notice them breaking (vs. full-fledge interop test). I did some interop testing recently between boringssl and openssl when I last updated the project in Dec 2022; but mainly using Kyber. We already provide a strong disclaimer in the README, we can add more details about the state of our interop testing.